pkg/controllers/job/job_controller_actions.go

```diff
@@ -584,6 +584,7 @@ func (cc *Controller) initJobStatus(job *batch.Job) (*batch.Job, error) {
 
 	job.Status.State.LastTransitionTime = metav1.Now()
 	job.Status.State.Phase = batch.Pending
+	job.Status.State.LastTransitionTime = metav1.Now()
```

Comment:
we have set it in L585

Reply:
I must forget release 0.4.1

Reply:
let's have a 0.4.1 release this month :)